tuf/ngclient/_internal/trusted_metadata_set.py

```diff
@@ -97,7 +97,7 @@ def __init__(self, root_data: bytes):
             RepositoryError: Metadata failed to load or verify. The actual
                 error type and content will contain more details.
         """
-        self._trusted_set = {}  # type: Dict[str: Metadata]
+        self._trusted_set = {}  # type: Dict[str, Metadata]
```

Comment:
```suggestion
        self._trusted_set: Dict[str, Metadata] = {}
```
Since we're fixing my broken annotations, let's do it like everywhere else :grimacing: